test/test_java_function.py

```diff
@@ -0,0 +1,26 @@
+import unittest2
+import main
+
+class TestJavaFunctionCalls(unittest2.TestCase):
```

Comment:
I like this pattern for testing. Would also be good to see some code to gast and gast to code tests if possible as well!